bindings/generator/templates/binding_web_meths.rs.j2

```diff
@@ -171,15 +171,15 @@ if {{ js_val }}.is_null() {
 }
 {%- elif type.kind == "i64_based" -%}
 {
-    let v = {{ js_val }}.dyn_into::<Number>().map_err(|_| TypeError::new("Not a number"))?.value_of();
+    let v = {{ js_val }}.dyn_into::<BigInt>().map_err(|_| TypeError::new("Not a number"))?.value_of();
     if v < (i64::MIN as f64) || (i64::MAX as f64) < v {
         return Err(JsValue::from(TypeError::new("Not an i64 number")));
     }
     v as i64
 }
 {%- elif type.kind == "u64_based" -%}
 {
-    let v = {{ js_val }}.dyn_into::<Number>().map_err(|_| TypeError::new("Not a number"))?.value_of();
+    let v = {{ js_val }}.dyn_into::<BigInt>().map_err(|_| TypeError::new("Not a number"))?.value_of();
     if v < (u64::MIN as f64) || (u64::MAX as f64) < v {
```

Comment:
Why `v` still `f64` ? :/

Reply:
I didn't go further because I was blocked by electron (cf #5379)

Reply:
if `value_of` converts to `f64` you will lose the information too

Reply:
> I didn't go further because I was blocked by electron (cf #5379)

Ok :+1:

Reply:
I did a quick change for web only ;)